### PR TITLE
adding 2020 wg reporting process steps

### PR DESCRIPTION
**Pull request by parispittman**
ref: https://github.com/kubernetes/steering/issues/164

### CODE REVIEW

**Comment by justaugustus** [comment]
/cc
(as @parispittman and @dims pinged me on it)

File: committee-steering/governance/annual-reports.md [hunk 1]
```diff
@@ -26,31 +26,41 @@ ensure that reports are complete, accurate, and submitted to the Steering
...
-guidance needed
-3. The March SIG Chairs and Tech Leads meetings will be used as follow up for
-the community groups that have questions from/to Steering and report
-retrospectives
-4. Steering will schedule private meetings to follow up on anything denoted as
+outlined for 2021. Working Groups, due to their temporary nature, will be the only group 
+to submit for 2020.   
+
+1. Steering Committee will assign one member as a liaison to each community group for kick 
+off, help, or guidance needed throughout the process
```
Comment (nikhita) [inline committee-steering/governance/annual-reports.md:35]: Can we mention here that we'll kick off the process on July 1st?

File: committee-steering/governance/annual-reports.md [hunk 2]
```diff
@@ -26,31 +26,41 @@ ensure that reports are complete, accurate, and submitted to the Steering
...
-reports are submitted so there is an established point of contact for any help or
-guidance needed
-3. The March SIG Chairs and Tech Leads meetings will be used as follow up for
-the community groups that have questions from/to Steering and report
-retrospectives
-4. Steering will schedule private meetings to follow up on anything denoted as
+outlined for 2021. Working Groups, due to their temporary nature, will be the only group 
+to submit for 2020.   
+
+1. Steering Committee will assign one member as a liaison to each community group for kick
```
Comment (nikhita) [inline committee-steering/governance/annual-reports.md:34]: Not for this PR, but commenting here to ensure we don't forget -- we should PR in the list of liaisons in the steering repo before July 1st (https://github.com/kubernetes/steering/issues/159).

File: committee-steering/governance/annual-reports.md [hunk 3]
```diff
@@ -26,31 +26,41 @@ ensure that reports are complete, accurate, and submitted to the Steering
...
+
+1. Steering Committee will assign one member as a liaison to each community group for kick 
+off, help, or guidance needed throughout the process
+2. Working Group Organizers create one report and send it to/share it with steering-private@kubernetes.io
+for review; reports should take ~30 days to complete (work with your liasion on timeline)
+3. Steering will schedule private meetings to follow up on anything denoted as
 a private matter on the report; reports are not made public until all private
 comments have been addressed
-5. Once reviewed and follow ups are completed, Steering will ask the chair(s)
+4. Once reviewed and follow ups are completed, the liasion will ask the chair(s)
```
Comment (nikhita) [inline committee-steering/governance/annual-reports.md:41]: nit: s/chair(s)/organizer(s)

File: committee-steering/governance/annual-reports.md (hunk 1 above)
Comment (dims) [reply committee-steering/governance/annual-reports.md:35]: +1

**Comment by dims** [comment]
looks good. thanks @parispittman

**Comment by jdumars** [comment]
not adding an official LGTM because it's steering, but I looked it over and it indeed LGTM :)

File: committee-steering/governance/annual-reports.md (hunk 2 above)
Comment (nikhita) [reply committee-steering/governance/annual-reports.md:34]: Created https://github.com/kubernetes/steering/pull/168

File: committee-steering/governance/annual-reports.md [hunk 4]
```diff
@@ -26,31 +26,41 @@ ensure that reports are complete, accurate, and submitted to the Steering
...
-the community groups that have questions from/to Steering and report
-retrospectives
-4. Steering will schedule private meetings to follow up on anything denoted as
+outlined for 2021. Working Groups, due to their temporary nature, will be the only group 
+to submit for 2020.   
+
+1. Steering Committee will assign one member as a liaison to each community group for kick 
+off, help, or guidance needed throughout the process
+2. Working Group Organizers create one report and send it to/share it with steering-private@kubernetes.io
+for review; reports should take ~30 days to complete (work with your liasion on timeline)
```
Comment (cblecker) [inline committee-steering/governance/annual-reports.md:37]:
```
committee-steering/governance/annual-reports.md:37:69: "liasion" is a misspelling of "liaison"
```

File: committee-steering/governance/annual-reports.md (hunk 3 above)
Comment (cblecker) [inline committee-steering/governance/annual-reports.md:41]:
```
committee-steering/governance/annual-reports.md:41:51: "liasion" is a misspelling of "liaison"
```

File: committee-steering/governance/annual-reports.md [hunk 5]
```diff
@@ -26,31 +26,41 @@ ensure that reports are complete, accurate, and submitted to the Steering
...
+
+### 2021 and on:
+1. Steering Committee will assign one member as a liaison to each community group for kick 
+off, help, or guidance needed throughout the process
+2. Chairs/Organizers create one report and send it to/share it with steering-private@kubernetes.io
+ for review; reports are due at the end of February 
+3. Steering will schedule private meetings to follow up on anything denoted as
+a private matter on the report; reports are not made public until all comments have 
+been addressed
+4. Once reviewed and follow ups are completed, the liasion will ask the chair(s)
```
Comment (cblecker) [inline committee-steering/governance/annual-reports.md:57]:
```
committee-steering/governance/annual-reports.md:57:51: "liasion" is a misspelling of "liaison"
```

**Pull request merged**